### DISCUSSION TITLE
Weird stuff with Reaper

### DISCUSSION THREAD

**Discussion by musicgreator** [apohl79/audiogridder#475] [Help]
On Windows 10.
When I put the AudioGridder Server App Link into the Windows dock, and then right click, I see all the recent Reaper sessions listed. As if AudioGridder thinks it is reaper. Also I noticed, when Server is running, when I click the docked Audiogridder icon, I get access to the opened Reaper Windows. Is this by design?

**Comment by apohl79** [comment]
Certainly not. I guess thats some local behavior. AG has no reference to reaper anywhere.

**Comment by musicgreator** [comment]
Forgot to mention, that reaper and the server are running on the same machine.

**Comment by musicgreator** [comment]
Is there any chance it is a Audiogridder bug, or can this only be an issue with my system?

**Reply by apohl79** [reply]
I don't know, on my windows VMs it does not happen. I can't say it is no bug in AG or JUCE, as I have no idea how you managed to do this.

**Reply by musicgreator** [reply]
Ok, it was a problem with my system which I have solved!
